File: perf.java
import java.util.concurrent.TimeUnit;

class perf {
    public static void main(String[] args) {
        long start = System.nanoTime();
        double sum = 0;

        int n = Integer.parseInt(args[0]);

        for (int i = 1; i <= n; ++i) {
            sum += Math.log(i) / Math.log(2);
        }

        double elapsed = (System.nanoTime() - start) / (double)TimeUnit.NANOSECONDS.convert(1, TimeUnit.SECONDS);

        System.out.println(String.format("Java: %.9fs\t%f", elapsed, sum));
    }
}

File: perf.py
# -*- encoding: utf-8 -*-
import sys
import time
import math

start = time.time()

n = int(sys.argv[1])

i = 1
total = 0
while i <= n:
    total += math.log(i, 2)
    i += 1

elapsed = time.time() - start

print("Python: {:.9f}s\t{}".format(elapsed, total))
